packages/lib-react-components/src/Media/Media.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import mime from 'mime-types'
+import Audio from './components/Audio'
+import ThumbnailImage from './components/ThumbnailImage'
+import Video from './components/Video'
+
+const DEFAULT_THUMBNAIL_DIMENSION = 999
+
+export default function Media(props) {
+  const mimeType = mime.lookup(props.src)
+
+  if (mimeType && mimeType.includes('image')) {
+    return (
+      <ThumbnailImage {...props} />
+    )
+  }
+
+  if (mimeType && mimeType.includes('video')) {
+    return (
+      <Video {...props} />
+    )
+  }
+
+  if (mimeType && mimeType.includes('audio')) {
+    return (
+      <Audio {...props} />
+    )
+  }
+
+  return null
+}
+
+Media.propTypes = {
```

Comment:
Most of these prop types are declared twice, once here and again in the component they're actually being used in. Given that this component is just a passthrough, they could either be removed, or exported from the components actually using them and composed together